draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2237,18 +2237,18 @@ relies on a random oracle. Applications using nonuniform encodings SHOULD carefu
 analyze the security implications of using such an encoding. When the required
 encoding is not clear, applications SHOULD use a random oracle.
 
-The below table lists the curves for which suites are defined and
-the subsection that gives the corresponding parameters.
-
-| E                         | Section              |
-|---------------------------|----------------------|
-| NIST P-256                | {{suites-p256}}      |
-| NIST P-384                | {{suites-p384}}      |
-| NIST P-521                | {{suites-p521}}      |
-| curve25519 / edwards25519 | {{suites-25519}}     |
-| curve448 / edwards448     | {{suites-448}}       |
-| secp256k1                 | {{suites-secp256k1}} |
-| BLS12-381                 | {{suites-bls12381}}  |
+The below table lists suites RECOMMENDED for some elliptic curves. The
```

Comment:
I'd swap the Section and Suites columns, too, to match this sentence. Namely, each table row identities a curve, the suite, and a pointer to more details.